bcos-ledger/src/libledger/Ledger.cpp

```diff
@@ -149,15 +149,17 @@ void Ledger::asyncPrewriteBlock(bcos::storage::StorageInterface::Ptr storage,
         return;
     }
     auto header = block->blockHeaderConst();
-
+    auto blockVersion = header->version();
     auto blockNumberStr = boost::lexical_cast<std::string>(header->number());
 
 
     size_t TOTAL_CALLBACK = 8;
-    if (writeTxsAndReceipts)
-    {  // 9 storage callbacks and write hash=>tx
-        TOTAL_CALLBACK = 9;
+    // if blockVersion >= 3.6, txs will be stored when commit block, not here
+    if (blockVersion >= uint32_t(bcos::protocol::BlockVersion::V3_6_VERSION))
+    {
+        --TOTAL_CALLBACK;
     }
+
```

Comment:
3.6 之前这个逻辑同样没走到，可以删除

```suggestion
    // if blockVersion >= 3.6, txs will be stored when commit block, not here
    if (blockVersion >= uint32_t(bcos::protocol::BlockVersion::V3_6_VERSION))
    {
        --TOTAL_CALLBACK;
    }

```

Reply:
这里没问题，3.6少一步，需要--